lib/annotations2triannon/open_annotation.rb

```diff
@@ -27,10 +27,7 @@ def initialize(graph=RDF::Graph.new, id=nil)
       else
         @graph = graph
         raise ArgumentError, 'graph must be an open annotation' unless is_annotation?
-        if id.nil?
-          @id = get_id
-        else
-        end
+        id.nil? ? @id = get_id : @id = id
```

Comment:
this is fine.  i'm wondering if this reads better:

@id = id.nil? ? get_id : id